.github/workflows/main.yml

```diff
@@ -17,8 +17,6 @@ jobs:
         apt update
         apt install -y meson sassc
     - name: Build
-      env:
```

Comment:
This is removed because `install_subdir` in `data/meson.build` only works with absolute paths:

```
ValueError: dst_dir must be absolute, got out/usr/local/share/themes/io.elementary.stylesheet.banana/gtk-3.0/assets
```

Reference: https://github.com/elementary/stylesheet/runs/2477411930